conda_build/environ.py

```diff
@@ -469,8 +469,9 @@ def linux_vars(compiler_vars, prefix, config):
     if config.arch == 32:
         compiler_vars['CFLAGS'] += ' -m32'
         compiler_vars['CXXFLAGS'] += ' -m32'
-    return {}
```

Comment:
Ah, OK. I agree is confusing. I would rather only return dicts instead of modifying in-place.